### ISSUE TITLE
Validation Loss Diverges

### ISSUE THREAD

**Issue by judithliatsf** [judithliatsf/to_distill_or_not#2]
Reason:
- Already overfit after 1st epoch: Change callbacks from `per epoch` to `per step`, or add more repeats
- Learning rate is too big: change `5e-4` to `2e-5`
